backend/app/models/skill.py

```diff
@@ -0,0 +1,8 @@
+from sqlalchemy import Column, Integer, String
+
+from app.db.base_class import Base
+
+
+class Skill(Base):
+    skill_id = Column(Integer, primary_key=True, index=True)
+    skill_name = Column(String, nullable=False)
```

Comment:
Can confirm if these fields are standardised against the fields in the DB schema?
https://lucid.app/lucidchart/3927e942-08c3-44fc-b056-e4117c97d0a0/edit?beaconFlowId=36667FEA57D405BF&invitationId=inv_b24547aa-e507-4177-9804-29fe5da262b4&page=0_0#